compiler/damlc/tests/daml-test-files/ProposalDSL.EXPECTED.rst

```diff
@@ -0,0 +1,33 @@
+
+.. _module-proposaldsl-55246:
+
+Module ProposalDSL
+------------------
+
+
+Templates
+^^^^^^^^^
+
+.. _template-proposaldsl-proposal-62786:
+
+template **Proposal**
```

Comment:
Should we show the type parameter here?

Reply:
Yep, but I left that for a separate PR, this one is just for recognizing generic templates (and non-generic templates under the new desugaring).

Reply:
:+1: